nemo_curator/utils/distributed_utils.py

```diff
@@ -106,6 +117,12 @@ def get_client(
     rmm_pool_size="1024M",
     enable_spilling=True,
     set_torch_to_use_rmm=False,
+    rmm_async=True,
+    rmm_maximum_pool_size=None,
+    rmm_managed_memory=False,
+    rmm_release_threshold=None,
+    rmm_track_allocations=False,
+    **cluster_kwargs,
```

Comment:
Can you also pass `cluster_kwargs` to the CPU `LocalCluster` path incase folks want to tweak those too? Update the docstring as well when you do to include that note. Unless there's a reason you only wanted to pass on them to GPU clusters.

Reply:
Works for me! Updated.